src/main/java/org/opentripplanner/routing/algorithm/Raptor.java

```diff
@@ -0,0 +1,307 @@
+package org.opentripplanner.routing.algorithm;
+
+import gnu.trove.iterator.TObjectIntIterator;
+import jersey.repackaged.com.google.common.collect.Sets;
+import org.joda.time.LocalDate;
+import org.opentripplanner.routing.edgetype.SimpleTransfer;
+import org.opentripplanner.routing.edgetype.TripPattern;
+import org.opentripplanner.routing.graph.Edge;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.trippattern.FrequencyEntry;
+import org.opentripplanner.routing.trippattern.TripTimeSubset;
+import org.opentripplanner.routing.vertextype.TransitStop;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/** Implements the RAPTOR algorithm; see http://research.microsoft.com/pubs/156567/raptor_alenex.pdf */
+public class Raptor {
```

Comment:
I believe this has been replaced with RaptorWorker. We should move the reference to the RAPTOR paper into the RaptorWorker Javadoc and remove this class unless there's some plan to use it elsewhere.